src/main/java/de/tum/in/www1/artemis/service/dataexport/DataExportCreationService.java

```diff
@@ -85,11 +89,18 @@ private DataExport createDataExportWithContent(DataExport dataExport) throws IOE
         dataExportExamCreationService.createExportForExams(userId, workingDirectory);
         dataExportCommunicationDataService.createCommunicationDataExport(userId, workingDirectory);
         addGeneralUserInformation(user, workingDirectory);
+        addArt15ReadmeFile(workingDirectory);
         var dataExportPath = createDataExportZipFile(user.getLogin(), workingDirectory);
         fileService.scheduleForDirectoryDeletion(workingDirectory, 30);
         return finishDataExportCreation(dataExport, dataExportPath);
     }
 
+    private void addArt15ReadmeFile(Path workingDirectory) throws IOException, URISyntaxException {
```

Comment:
```suggestion
    private void addReadmeFile(Path workingDirectory) throws IOException, URISyntaxException {
```
I assume that (nearly) no developer will have an idea what Art. 15 is without further context. Just calling it a readme should be clear enough.